app/src/sharedTest/java/org/oppia/app/topic/info/TopicInfoFragmentTest.kt

```diff
@@ -132,6 +169,22 @@ class TopicInfoFragmentTest {
     return ActivityScenario.launch(intent)
   }
 
+  private fun setTextInTextView(value: String): ViewAction {
```

Comment:
You can make this code more readable:
```
  private fun setTextInTextView(value: String): ViewAction {
    return object : ViewAction {
      override fun getConstraints(): Matcher<View> {
        return allOf(isDisplayed(), isAssignableFrom(TextView::class.java))
      }

      override fun perform(uiController: UiController, view: View) {
        (view as TextView).text = value
      }

      override fun getDescription(): String {
        return "replace text"
      }
    }
  }
```

Reply:
@rt4914 Can I add a comment above stating the use of the function, or what else do you suggest?

Reply:
Actually thats fine here as the function name is very clear and this function is private also.

Reply:
Okay. As I had added in the second commit, you review other changes and then I will remove the comment. Thanks.